packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/e2e/pageObject.ts

```diff
@@ -0,0 +1,314 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {isUaSearchEvent} from '../../../../../../playwright/utils/requests';
+
+const breadcrumbElementsSelectors = {
+  regularFacet: {
+    component: 'c-quantic-facet',
+    facetValueComponent: 'c-quantic-facet-value',
+    breadcrumbElementTestId: 'facet-breadcrumb',
+    breadcrumbValueElementTestId: 'facet-breadcrumb-value',
+  },
+  numericFacet: {
+    component: 'c-quantic-numeric-facet',
+    facetValueComponent: 'c-quantic-facet-value',
+    breadcrumbElementTestId: 'numeric-facet-breadcrumb',
+    breadcrumbValueElementTestId: 'numeric-facet-breadcrumb-value',
+  },
+  categoryFacet: {
+    component: 'c-quantic-category-facet',
+    facetValueComponent: 'c-quantic-category-facet-value',
+    breadcrumbElementTestId: 'category-facet-breadcrumb',
+    breadcrumbValueElementTestId: 'category-facet-breadcrumb-value',
+  },
+  timeframeFacet: {
+    component: 'c-quantic-timeframe-facet',
+    facetValueComponent: 'c-quantic-facet-value',
+    breadcrumbElementTestId: 'date-facet-breadcrumb',
+    breadcrumbValueElementTestId: 'date-facet-breadcrumb-value',
+  },
+};
+
+export class BreadcrumbManagerObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  async countAllFacetsBreadcrumb() {
+    return (
+      (await this.allRegularFacetBreadcrumb.count()) +
+      (await this.allNumericFacetBreadcrumb.count()) +
+      (await this.allCategoryFacetBreadcrumb.count()) +
+      (await this.allTimeframeFacetBreadcrumb.count())
+    );
+  }
+
+  /** REGULAR FACET */
+  get allRegularFacetBreadcrumb(): Locator {
+    return this.page.getByTestId(
+      breadcrumbElementsSelectors.regularFacet.breadcrumbElementTestId
+    );
+  }
+
+  getAllRegularFacetBreadcrumbValuesForFacet(
+    regularFacetBreadcrumb: Locator
+  ): Locator {
+    return regularFacetBreadcrumb.getByTestId(
+      breadcrumbElementsSelectors.regularFacet.breadcrumbValueElementTestId
+    );
+  }
```

Comment:
these methods dont seem to be used anywhere? am I missing something?

Reply:
True I think those were stuff I assumed I would need and ended up not? ty